SPECS/kata-containers/kata-osbuilder.sh

```diff
@@ -245,8 +245,8 @@ update_config()
     local image_osbuilder_dir="${IMAGE_TOPDIR}/osbuilder-images"
     local image_dir="${image_osbuilder_dir}/$KVERSION"
     local initrd_dest_path="${image_dir}/${DISTRO}-kata-${KVERSION}.initrd"
-    sed -i -e "s|kernel = \"/usr/share/kata-containers/vmlinux.container\"|kernel = \"/boot/vmlinux.bin\"|" /usr/share/defaults/kata-containers/configuration.toml
-    sed -i -e "s|image = \"/usr/share/kata-containers/kata-containers.img\"|initrd = \"$initrd_dest_path\"|" /usr/share/defaults/kata-containers/configuration.toml
+    sed -i -e "s|kernel = .*$|kernel = \"/boot/vmlinux.bin\"|" /usr/share/defaults/kata-containers/configuration.toml
```

Comment:
I am surprised that we don't use these paths:

/var/cache/kata-containers/vmlinuz.container
/var/cache/kata-containers/kata-containers-initrd.img

But, it's OK with me if you want to or have to keep the other paths for now.

Reply:
/var/cache/kata-containers/vmlinuz.container and /var/cache/kata-containers/kata-containers-initrd.img are symlinks. I found it better to have the actual contents of the symlink be populated in the config, as it makes debugging easier.